internal/featuresupport/features_misc.go

```diff
@@ -281,16 +281,14 @@ func (f *skipMcoReboot) getIncompatibleFeatures(openshiftVersion string) *[]mode
 }
 
 func (f *skipMcoReboot) getIncompatibleArchitectures(openshiftVersion *string) *[]models.ArchitectureSupportLevelID {
```

Comment:
why is that function needed if it just return nil?

Reply:
It is part of an interface

Reply:
how can it be? it's an internal function

Reply:
https://github.com/openshift/assisted-service/blob/e56197390fd182c3e43682dd3c9d5c74e7b206bc/internal/featuresupport/support_level_feature.go#L8